Source/Charts/Charts/CombinedChartView.swift

```diff
@@ -55,7 +55,7 @@ open class CombinedChartView: BarLineChartViewBase, CombinedChartDataProvider
             
             self.highlighter = CombinedHighlighter(chart: self, barDataProvider: self)
             
-            (renderer as! CombinedChartRenderer?)!.createRenderers()
+            (renderer as? CombinedChartRenderer)?.createRenderers()
```

Comment:
old code is a bit weird (maybe the compiler bug). But if it's safe unwrap here, why not use `(renderer as! CombinedChartRenderer).createRenderers()`? `renderer` has been setup as combined chart renderer in `initialize()` above.

Reply:
I'd rather not rely on the assumption `renderer !== nil`

Reply:
I agree with @russellbstephens . However I think a future ticket should involve making `renderer` a required property since you can't do anything with a chart without a renderer.